src/taskList.js

```diff
@@ -0,0 +1,56 @@
+import Task from './task.js';
+
+export default class TaskList {
+  constructor(tasks) {
+    this.tasks = tasks.map((el) => new Task(el.description, el.completed, el.index));

```

Comment:
You are using a class inside another class. I think you can avoid using Task class as I mentioned in another comment.

Reply:
Nested class replaces with an object.